docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/movement/col2im-15.rst

```diff
@@ -0,0 +1,216 @@
+  * **Range of values**: 1D tensor of two positive integer numbers (height and width)
+  * **Type**: *T_IND*
+
+* **3**: *kernel_size*
+
+  * **Description**: size of the sliding blocks. **Required.**
+  * **Range of values**: 1D tensor of non-negative integer numbers
+  * **Type**: *T_IND*
+
+**Outputs**
+
+* **1**: The output tensor the output image of type *T* and shape:
+
+  * ``(N, C, output_size[0], output_size[1])`` in case of batched input,
+  * ``(C, output_size[0], output_size[1])`` in case of non-batched input.
+
+**Types**
+
+* *T*: any supported data type.
+* *T_IND*: ``int64`` or ``int32``.
+
+**Examples**
+
+All examples assume ``C = 3``.
+
+*Example 1: default optional Parameters*
+
+For inputs ``output_size`` = [16, 16] and ``kernel_size`` = [2, 2]
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="Col2Im" ... >
+        <input>
+            <port id="0" precision="I32">
+                <dim>3</dim>     <!-- batch_axis -->
+                <dim>12</dim>    <!-- C * Product(kernel_size) -->
+                <dim>225</dim>   <!-- L -->
+            </port>
+            <port id="1" precision="I32">
+                <dim>2</dim>     <!-- output_size -->
+            </port>
+            <port id="2" precision="I32">
+                <dim>2</dim>     <!-- kernel_size -->
+            </port>
+        </input>
+        <output>
+            <port id="1" precision="I32">
+                <dim>3</dim>     <!-- batch_axis -->
+                <dim>3</dim>     <!-- C -->
+                <dim>16</dim>    <!-- output_size[0] -->
+                <dim>16</dim>    <!-- output_size[1] -->
+            </port>
+        </output>
+    </layer>
+
+
+*Example 2: non-default dilations, padding and strides*
+
+For inputs ``output_size`` = [16, 16] and ``kernel_size`` = [3, 3]
+
+.. code-block:: xml
+   :force:
+
+    <layer ... type="Col2Im" ... >
+        <data dilations="2,2" pads_begin="1,1" pads_end="1,1" strides="2,2"/>
+        <input>
+            <port id="0" precision="I32">
+                <dim>1</dim>     <!-- batch_axis -->
+                <dim>27/dim>     <!-- C * Product(kernel_size) -->
+                <dim>25</dim>    <!-- L -->
+            </port>
+            <port id="1" precision="I32">
+                <dim>2</dim>     <!-- output_size -->
+            </port>
+            <port id="2" precision="I32">
+                <dim>2</dim>     <!-- kernel_size -->
+            </port>
+        </input>
+        <output>
+            <port id="1" precision="I32">
+                <dim>1</dim>     <!-- batch_axis -->
+                <dim>3</dim>     <!-- C -->
+                <dim>16</dim>    <!-- output_size[0] -->
+                <dim>16</dim>    <!-- output_size[1] -->
+            </port>
+        </output>
+    </layer>
+
+*Example 3: non-default dilations and padding*
+
+For inputs ``output_size`` = [32, 32] and ``kernel_size`` = [2, 2]
```

Comment:
For `Example 3` and `Example 4`, the ports for `output_size` and `kernel_size` inputs are still missing.

Reply:
I forgot to commit it :) It's fixed.